README-dfds-data.md

```diff
@@ -17,7 +17,18 @@ You must log in with the shared credentials. They are found
 ## Adding and deploying dashboards
 
 Dashboards should be created/modified in the UI, then saved as JSON and put in the dashboards folder
-(/grafana/dashboards). The pipeline will convert them to configmaps and they should be persisted in
-k8s this way. The JSON dashboards are converted to configmaps with a
+(/grafana/dashboards). You can save the dashboard as JSON in the UI by going to settings and copying
```

Comment:
I guess that the json contain the configuration of the dashboard and not the actually data?
Perhaps... `You can save the configuration of the dashboard as JSON...`